api/src/test/java/ca/bc/gov/educ/api/edx/controller/SecureExchangeNoteControllerTest.java

```diff
@@ -139,6 +138,27 @@ public void testGetNotes_GivenValidExchangeIdWithNoNotes_ExpectReturns204NoConte
                 .andDo(print()).andExpect(status().isNoContent());
     }
 
+    @Test
+    @Transactional
+    public void testDeleteNotes_GivenValidExchangeId_ExpectReturns204NoContentWithoutNotes() throws Exception {
+        ObjectMapper objectMapper1 = new ObjectMapper();
+        SecureExchangeEntity entity = this.secureExchangeRequestRepository.save(exchangeMapper.toModel(this.getSecureExchangeEntityFromJsonString()));
+        SecureExchangeNoteEntity note = noteMapper.toModel(objectMapper1.readValue(
+                this.createDummyNoteJsonWithAuditColumns(entity.getSecureExchangeID().toString(), "Test", "TOSSE", "2020-02-09T00:00:00"),
+                SecureExchangeNote.class
+        ));
+        note.setSecureExchangeEntity(entity);
+        SecureExchangeNoteEntity noteEntity = this.secureExchangeRequestNoteRepository.save(note);
+        entity.setSecureExchangeNotes(new HashSet<>());
+        entity.getSecureExchangeNotes().add(note);
+        this.secureExchangeRequestRepository.save(entity);
+
+        this.mockMvc.perform(delete(URL.BASE_URL_SECURE_EXCHANGE+"/" +URL.SECURE_EXCHANGE_ID_NOTES + "/{secureExchangeNoteId}", entity.getSecureExchangeID().toString(), noteEntity.getSecureExchangeNoteID().toString())
+                        .with(jwt().jwt((jwt) -> jwt.claim("scope", "WRITE_SECURE_EXCHANGE"))))
```

Comment:
will need to change this to DELETE_SCHOOL_NOTE

Reply:
After changing the scope this test fails with a 403 insufficient_scope. I tried adding the OpenAPIDefinition declaration to the NoteEndpoint class as well as adding this scope to the ExchangeEndpoint tag. But that has not helped, I am not sure what i am missing, but i bet you do.